modules/text2vec-jinaai/config.go

```diff
@@ -27,7 +27,6 @@ func (m *JinaAIModule) ClassConfigDefaults() map[string]interface{} {
 		"vectorizeClassName": ent.DefaultVectorizeClassName,
 		"model":              ent.DefaultJinaAIModel,
 		"baseURL":            ent.DefaultBaseURL,
-		"dimensions":         &ent.DefaultDimensions,
```

Comment:
is dimension not a parameter here?

Reply:
I double checked it and I have removed it from here bc there is no default `dimensions` setting, bc this setting wasn't applicable to some model, that's why it was removed